bitnami/mongodb/values.yaml

```diff
@@ -130,7 +130,7 @@ tls:
 
   ## Generate a custom CA and self-signed certificates.
   ##
-  autoGenerated: false
+  autoGenerated: true
```

Comment:
Why we are setting up autoGenerated to true by default?

Reply:
Sorry, I used the wrong base branch which included changes to mongoDB, please ignore this.

The issue should be fixed now.